### ISSUE TITLE
accessibility examples are not working with screen readers in chrome

### ISSUE THREAD

**Issue by becka11y** [fusioncharts/fusioncharts-dist#59]
I have tried a few different examples at: https://www.fusioncharts.com/extensions-accessibility/.  They are not working with JAWS nor NVDA in chrome browser. I do believe they used to work correctly.  
I have specifically tried 

- multi-series-column
- mscombidy2d.html
- multi-series-column.html
- single-series-bar.html
JAWS in Chrome will announce the chart name and axis but it does not announce the values as I arrow through them. I can see the focus moving but there is no announcement. I can tab into the legend and it announces.  This used to work???

We are also using the multi-column-bar within a product and it is not announcing there, either.  But, we have a column chart that IS working correctly.  

Note that the charts work and announce correctly when using NVDA or JAWS in Firefox.  It is also working correctly in VoiceOver in Safari.

**Comment by becka11y** [comment]
this appears to be solved with an update to Chrome.  I am not seeing the issues using Version 117.0.5938.63 (Official Build) (64-bit)

**Comment by becka11y** [comment]
closing
